sql/core/src/main/scala/org/apache/spark/sql/execution/QueryExecution.scala

```diff
@@ -92,6 +92,8 @@ class QueryExecution(
    * accessing after iteration. (Calling `collect()` is one of known bad usage.)
    * If you want to store these rows into collection, please apply some converter or copy row
    * which produces new object per iteration.
+   * Given QueryExecution is not a public class, end users are discouraged to use this: please
+   * user `Dataset.rdd` instead which conversion will be applied.
```

Comment:
user -> use
which -> in which
or
which -> where  ?

Reply:
Nice finding. Applied.